src/cf.data.pre

```diff
@@ -4686,7 +4686,13 @@ DOC_START
 		sn	Unique sequence number per log line entry
 		err_code    The ID of an error response served by Squid or
 				a similar internal error identifier.
-		err_detail  Additional err_code-dependent error information.
+
+		err_detail  Additional err_code-dependent error information. Multiple
+			details are separated by the plus sign ('+'). Admins should not
```

Comment:
I am not sure the `+` sign is the best choice here. On one hand, that is what some OpenSSL errors are using, and some admin scripts might be able to handle that delimiter better already. On the other hand, we normally use commas for lists, and using `+` might make it more difficult to distinguish a group of OpenSSL error details from other error details. Which delimiter do _you_ consider the best and why?

Reply:
IMO the syntax (whatever it is or becomes) should be the same as Squid will use in the `Proxy-Status:` header `detail=` parameter defined by RFC 9209 section 2.1.5.